addons/website_slides/tests/test_review.py

```diff
@@ -0,0 +1,41 @@
+from odoo.addons.website_slides.tests import common
+from odoo.tools.misc import mute_logger
+from odoo.tests import tagged, HttpCase
+from odoo import http
+
+@tagged('functional')
```

Comment:
it seems a bit overkill to have a new file just for that test, can we put it in an existing class / file ? (that would make sens)